docs/src/use/configure/migration-guide.md

```diff
@@ -27,7 +27,6 @@ To use flat config with ESLint v8, place a `eslint.config.js` file in the root o
 While the configuration file format has changed from eslintrc to flat config, the following has stayed the same:
 
 * Syntax for configuring rules
-* Syntax for configuring processors
```

Comment:
This should not be removed. The syntax for configuring processors is the same (using `processor`).

Reply:
It hasn't stayed the _same_ though, it's expanded. How about if I modified to `Syntax for configuring processors by their name`?

Reply:
That's a detail that isn't important in the migration guide. What we're saying here is that the same way you specified a processor in eslintrc will work in flat config. The goal of this guide is to help people migrate their existing eslintrc configs, so we should tailor all of the information for that purpose. We have a larger config file guide that explores more of the available options.